packages/js/product-editor/src/components/variations-table/variation-actions-menus/variation-quick-update-menu-item.tsx

```diff
@@ -95,7 +96,11 @@ VariationQuickUpdateMenuItem.Slot = ( {
 					return null;
 				}
 
-				return <MenuGroup>{ sortFillsByOrder( fills ) }</MenuGroup>;
+				return renderGroup ? (
```

Comment:
This feels odd. Why would `renderGroup === true` be used as opposed to using a `VariationQuickUpdateMenuGroup`?